daisy/step_create_snapshots.go

```diff
@@ -48,10 +49,16 @@ func (c *CreateSnapshots) run(ctx context.Context, s *Step) DError {
 		// Get source disk link if SourceDisk is a daisy reference to a disk.
 		if d, ok := w.disks.get(ss.SourceDisk); ok {
 			ss.SourceDisk = d.link
+
+			// Override snapshot link due that disk may be from a different project
+			if ss.Project != d.Project {
+				ss.link = fmt.Sprintf("projects/%s/global/snapshots/%s", d.Project, ss.Name)
```

Comment:
Can snapshots be zonal?

Reply:
No per my underadanding: https://cloud.google.com/compute/docs/regions-zones/global-regional-zonal-resources.